### DISCUSSION TITLE
iPadOS 14 “control + s” causing blink to freeze in SSH

### DISCUSSION THREAD

**Discussion by toddsunpark** [blinksh/blink#1934] [Support] [answered]
### Checklist

- [ x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [ x] Read the [docs](http://blink.sh/docs/)
- [ x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration
Blink Version 13.5.7.224
iPadOS 14

### Describe the bug 

When using SSH the use of “control+s” keystroke appears to cause Blink to lock up and become unresponsive.  In order to get it working again you have to kill the app and start over.  I’ve tested on multiple SSH connections.

In previous versions of iPadOS the “control+s” would pass through to the TTY session.  I use it constantly with midnight commander to perform quick searches.

**Comment by javierdemartin** [comment]
I am not able to reproduce that, SSH'ing into a server and pressing `control + S` does not freeze the session. Do you have the `control` key mapped to other key inside the Settings > Keyboard page or any other configuration related to the `control` key?

**Comment by toddsunpark** [comment]
Let me try this again.

**Comment by willhbr** [comment, marked as answer]
This is working as intended - Control-S locks the terminal, Control-Q unlocks it.

More info and how to disable it: https://unix.stackexchange.com/questions/12107/how-to-unfreeze-after-accidentally-pressing-ctrl-s-in-a-terminal#12108

**Comment by toddsunpark** [comment]
This is an interesting little piece of history that I was unaware of.
